scripts/fetch-aar.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+#
+# Convenience script to fetch the AAR and validate its digital signature
+# when you don't want to download it automatically using gradle.
+#
+
+set -e
+if [ $# -ne 1 ]; then
+    echo "usage: $0 version" 1>&2
+    exit 1
+fi
+rm -rf libs/*.aar
```

Comment:
We should probably delete `android-libs-$version.aar` only, so wget does not clobber it. Or we can configure wget not to create backup files and directly overwrite them if they're already there (<- this seems more idempotent)